camelot/admin/object_admin.py

```diff
@@ -694,7 +694,7 @@ def _expand_field_attributes(self, field_attributes, field_name):
             #
             direction = field_attributes.get('direction', 'onetomany')
             if direction.endswith('many') and related_admin:
-                field_attributes['columns'] = related_admin.get_columns()
+                field_attributes['columns'] = [(field, related_admin.get_field_attributes(field)) for field in related_admin.get_columns()]
```

Comment:
ik denk niet dat dit nodig is, die columns komen uiteindelijk ook in de 'add_columns' terecht.

dus ik denk dat de 'set_columns' van de OneToManyEditor kan aangepast worden om die field attributes ook weg te laten.